squad_client/commands/download_attachments.py

```diff
@@ -0,0 +1,86 @@
+from squad_client import logging
+from squad_client.core.command import SquadClientCommand
+from squad_client.core.models import Squad, TestRun
+from squad_client.shortcuts import get_build
+
+
+logger = logging.getLogger(__name__)
+
+
+class DownloadAttachmentsCommand(SquadClientCommand):
+    command = "download_attachments"
+    help_text = "download the attachments from a SQUAD testrun"
+
+    def register(self, subparser):
+        parser = super(DownloadAttachmentsCommand, self).register(subparser)
+        parser.add_argument(
+            "--group",
+            help="SQUAD group",
+            required=True
+        )
+        parser.add_argument(
+            "--project",
+            help="SQUAD project",
+            required=True
+        )
+        parser.add_argument(
+            "--build",
+            help="Build version. Examples: my-build-version. Or pre-defined build aliases: latest and latest-finished",
+            required=True,
+        )
```

Comment:
Since the user will already have testrun id in hand, I'd say `group`, `project` and `build` aren't necessary to be provided. `testrun` and `filenames` should be enough. What do you think?

Reply:
Good point! Removed :)